contracts/MemeBase.sol

```diff
@@ -84,61 +84,41 @@ contract MemeBase is MemeFactory {
     /// @dev MemeBase constructor
     constructor(
         address _olas,
-        address _usdc,
+        address _weth,
         address _router,
         address _factory,
         uint256 _minNativeTokenValue,
-        address _weth,
         address _l2TokenRelayer,
-        address _oracle,
         address _balancerVault,
         bytes32 _balancerPoolId
-    ) MemeFactory(_olas, _usdc, _router, _factory, _minNativeTokenValue) {
-        weth = _weth;
+    ) MemeFactory(_olas, _weth, _router, _factory, _minNativeTokenValue) {
```

Comment:
On Celo we don't have weth so not sure why we're making this part of MemeFactory now

Reply:
It was mistyped, it's meant to be a nativeToken, will fix naming

Reply:
Ah but it's correct then. WETH is nativeToken for MemeBase.